src/plone/restapi/tests/test_users.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.app.testing import setRoles
+from plone.app.testing import SITE_OWNER_NAME
+from plone.app.testing import SITE_OWNER_PASSWORD
+from plone.app.testing import TEST_USER_ID
+from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+from plone.restapi.testing import RelativeSession
+
+import transaction
+import unittest
+
+
+class TestUsersEndpoint(unittest.TestCase):
+
+    layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.app = self.layer['app']
+        self.portal = self.layer['portal']
+        self.portal_url = self.portal.absolute_url()
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+
+        self.api_session = RelativeSession(self.portal_url)
+        self.api_session.headers.update({'Accept': 'application/json'})
+        self.api_session.auth = (SITE_OWNER_NAME, SITE_OWNER_PASSWORD)
+
+        properties = {
+            'email': 'noam.chomsky@mit.edu',
+            'username': 'noamchomsky',
+            'fullname': 'Noam Avram Chomsky',
+            'home_page': 'web.mit.edu/chomsky',
+            'description': 'Professor of Linguistics',
+            'location': 'Cambridge, MA'
+        }
+        api.user.create(
+            email='noam.chomsky@mit.edu',
+            username='noam',
+            properties=properties
+        )
+        transaction.commit()
+
+    def test_list_users(self):
+        response = self.api_session.get('/@users')
+
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(3, len(response.json()))
+        user_ids = [user['id'] for user in response.json()]
+        self.assertTrue('admin' in user_ids)
```

Comment:
Use `assertIn('admin', user_ids)` -> better assertion failure message than `False is not True` 😉